middlewares/authenticate.js

```diff
@@ -79,13 +78,13 @@ module.exports = async (req, res, next) => {
         });
 
         // add user data to `req.userData` for further use
-        req.userData = await users.fetchUser({ userId });
-
+        req.userData = await dataAccess.retrieveUsers({ id: userId });
         return checkRestricted(req, res, next);
       } else {
         return res.boom.unauthorized("Unauthenticated User");
       }
     } else {
+      
```

Comment:
can you remove the trailing spaces here. Github is throwing warning 

![image](https://github.com/Real-Dev-Squad/website-backend/assets/97341921/028c5d95-7a6c-4af7-a723-9bfac0a21018)